api/test/wfl/unit/logging_test.clj

```diff
@@ -25,6 +26,19 @@
     (and (= (:severity json) (-> severity name str/upper-case))
          (boolean (check-message? test (get-in json [:message :result]))))))
 
+(deftest endpoint
```

Comment:
Thanks for adding this, with your instructions I see it pass locally:

```
wm111-e35:wfl okotsopo$ cd api; WFL_WFL_URL=http://localhost:3000 clojure -M:test --focus wfl.unit.logging-test/endpoint
WARNING: Specified path is external to project: ../derived/api/test/resources
WARNING: Specified path is external to project: ../derived/api/src
WARNING: Specified path is external to project: ../derived/api/resources
--- unit (clojure.test) ---------------------------
wfl.unit.logging-test
  endpoint
    the /logging_level endpoint works

1 tests, 3 assertions, 0 failures.
```

Since this relies on a WFL server I would make this a system test instead.  That also gets around the whole "GitHub automated release action won't complete due to a failing automated test" issue.

Reply:
Good idea.